src/fulltrade/TradePanel.js

```diff
@@ -358,12 +373,13 @@ export default class TradePanel extends Component {
         const receipt = trade.receipt;
         const isTick = trade.durationUnit && trade.durationUnit.slice(-1) === 't';
         const isBelow2Min = isTick || durationToSecs(trade.duration, trade.durationUnit) < 120;
-        const isIntraDay = durationToSecs(trade.duration, trade.durationUnit) <= 86400;
+        const isIntraDay = durationToSecs(trade.duration, trade.durationUnit) < 86400;
```

Comment:
This is highly context specific, but ok, since we already have durationToSecs which is also very context specific =P